### PR TITLE
Align folder structure with sidebar.js

### PR DESCRIPTION
**Pull request by pchico83**
Aligning URLs and folder structure improves SEO and simplifies development

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -1,21 +1,106 @@
+[[redirects]]
+  from = "/docs/1.19/get-started/overview"
```
Comment (pchico83) [inline netlify.toml:2]: The 1.19 prefix should be removed once 1.19 is the default version

**Review by ifbyol — COMMENTED** [review]
Don't we need the 4 files deleted?

**Comment by rlamana** [comment]
What do we currently do to check possible 404s or missing redirections?

**Comment by pchico83** [comment]
> What do we currently do to check possible 404s or missing redirections?

@rlamana I am doing it manually by checking all files that changed their location. After merge, we can monitor 404s

**Comment by pchico83** [comment]
> Don't we need the 4 files deleted?

@ifbyol These are the deleted files:
- src/content/core/components/development-containers.mdx
- src/content/get-started/cloud-provider-guides/index.mdx
- src/content/guides/samples/more.mdx
- src/content/manifest/hybrid-development.mdx
AFAIK, we aren't linking to them from anywhere, but I have added redirects just in case

**Review by ifbyol — APPROVED** [review]
Changes looks good and make sense to align folder structure with the URLs. It is a bit hard to review all links one by one but I was doing a general pass to the docs and everything looks good!

File: src/content/core/okteto-variables.mdx [hunk 2]
```diff
@@ -196,15 +196,15 @@ USERNAME has a static value, and PASSWORD is dynamically generated.
...
-- `OKTETO_HOME`: 
-- `OKTETO_FORCE_REMOTE`: 
-- `OKTETO_FOLDER`: 
-- `OKTETO_REMOTE_CLI_IMAGE`: 
+- `OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED`:
+- `OKTETO_KUBERNETES_TIMEOUT`:
+- `OKTETO_HOME`:
+- `OKTETO_FORCE_REMOTE`:
+- `OKTETO_FOLDER`:
+- `OKTETO_REMOTE_CLI_IMAGE`:
```
[the remark this answers is not in the document]
Comment (pchico83) [reply src/content/core/okteto-variables.mdx:204]: Right. It's coming from an incomplete merge, but we need to send a PR to fix it

**Pull request merged**